src/appcontroller.cpp

```diff
@@ -168,7 +166,7 @@ int ApplicationController::UserRunInstallerCallback(const wchar_t* filePath)
     return ms_cbUserRunInstaller(filePath);
 }
 
-int ApplicationController::AlternateAppcastCallback(bool manual, struct Appcast& appcast)
+int ApplicationController::AlternateAppcastCallback(bool manual, struct Appcast& appcast, bool *silent)
```

Comment:
I think the `silent` flag belongs in `Appcast` and that would make this parameter list cleaner.